src/debug/createdump/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+project(createdump)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Set the RPATH of sos so that it can find dependencies without needing to set LD_LIBRARY_PATH
```

Comment:
A nit: should be Set the RPATH of createdump ...
But actually, why does the createdump need RPATH?